js-md5/js-md5-tests.ts

```diff
@@ -1,20 +1,29 @@
+import * as md5 from "js-md5";
```

Comment:
Absolutely. Should I do something with the version number too? It looks like the travis build doesn't like it for some reason.